docs/security/unauthorized-sign-in.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Unauthorized sign-in
+description: Notify users of unauthorized sign-ins to their accounts
+---
+
+Clerk identifies attempts by unauthorized individuals or entities to access a user's account.
+
+## Detecting unauthorized sign-ins
+
+Whenever a sign-in attempt is completed from an unfamiliar device, Clerk sends an email notification to the account owner with information about the newly created session.
+
+The account owner can then choose to immediately revoke this session.
+
+To configure this feature:
+
+1. In the Clerk Dashboard, navigate to the [**Emails**](https://dashboard.clerk.com/last-active?path=customization/email) page.
+1. Select the **New Client Sign In** option. You'll be redirected to the template settings page.
+1. Toggle **Delivered by Clerk** to enable the feature.
```

Comment:
This will change soon. The correct way to enable/disable the feature needs an Emails page redesign which should be landing soonish.

I won't merge this PR before the new redesign is live and I update this step.